config/armadactl/config.yaml

```diff
@@ -0,0 +1,4 @@
+target: "server:50051"
```

Comment:
These configs won't be able to stay in the repo. We probably want to allow localdev to switch between them either using:

- Env Vars like in https://github.com/armadaproject/armada/tree/master/developer/env

- Extra configs like in https://github.com/armadaproject/armada/tree/master/developer/config

Happy to assist in this if you would like.

Reply:
That's fine, but beyond the scope of this PR I think.